addons/t9n/static/src/core/project_list.js

```diff
@@ -0,0 +1,87 @@
+import { Component, useState, onMounted } from "@odoo/owl";
+import { useService } from "@web/core/utils/hooks";
+import { formatList } from "@web/core/l10n/utils";
```

Comment:
I usually order the imports alphabetically with a newline between each module:
```suggestion
import { Component, onMounted, useState } from "@odoo/owl";

import { formatList } from "@web/core/l10n/utils";
import { useService } from "@web/core/utils/hooks";